c2corg_ui/templates/outing/edit.html

```diff
@@ -563,14 +567,14 @@ <h5 translate ng-show="outing.associations.users.length == 0">You can look up fo
 
       </div>
 
+      ## FIXME: really useful?
       <div class="step step-4"></div>
```

Comment:
well, if it is empty then hell no! just delete it and rename `step-5` to `step-4`